services/alertmanager/alertmanager.go

```diff
@@ -730,10 +732,13 @@ func (svc *Service) SendAlerts(ctx context.Context, alerts ammodels.PostableAler
 }
 
 // GetAlerts returns alerts available in alertmanager.
-func (svc *Service) GetAlerts(ctx context.Context) ([]*ammodels.GettableAlert, error) {
-	resp, err := amclient.Default.Alert.GetAlerts(&alert.GetAlertsParams{
-		Context: ctx,
-	})
+func (svc *Service) GetAlerts(ctx context.Context, params *alert.GetAlertsParams) ([]*ammodels.GettableAlert, error) {
```

Comment:
Why to not accept here `params *services.FilterParams` ?

Reply:
Looks like i misunderstood how the alertmanager <> services dependencies is supposed to work, I've updated it (and the tests too).